econml/orf/_ortho_forest.py

```diff
@@ -248,6 +248,8 @@ def __init__(self,
         self.verbose = verbose
         self.batch_size = batch_size
         self.categories = categories
+        if self.categories != 'auto':
+            self.categories = [self.categories]  # OneHotEncoder expects a 2D array with features per column
```

Comment:
But on a related note it also looks like there's an existing bug in those `fit` methods because they are reassiging `self.categories` to the list-wrapped version if it's not `'auto'`, which means that multiple calls to fit will wrap the list multiple times and therefore will break.  Could you add a test that calls `fit` more than once on those OrthoForest flavors with non-default categories, and make sure that it passes?

Reply:
The special setter for `categories` in `TreatmentExpansionMixin` fixed this issue.